app/swagger/swagger/requests/profile.rb

```diff
@@ -401,7 +401,6 @@ class Profile
             key :description, 'Response is OK'
             schema do
               key :type, :object
-              property(:validation_key, type: :integer)
```

Comment:
No longer used. Validation key used to be used in the post request [here](https://github.com/department-of-veterans-affairs/vets-api/pull/3441/files#diff-ffc28295b605c141000d0b1ea9562d69e67efd161cec9c8ae4e934364b1a9991R15). This was removed years ago. Not sure how the tests were passing still